rasa/nlu/utils/__init__.py

```diff
@@ -51,6 +55,7 @@ def list_directory(path: Text) -> List[Text]:
         results = []
         for base, dirs, files in os.walk(path):
```

Comment:
are files here in random order? aren't they are ordered according to system order?